chingu-voyages/v50-tier2-team-14
# Fix/moved images from public to src

# Description
This PR is to check whether autodeployment on render was affected by our images stored in public folder. Assuming that images were not being processed by webpack when they were stored in public, I moved them to src. We will find out whether this theory is true after we merge this PR to development. 


## Main changes explained:
- Moved images from public to src 
- Updated image imports in relevant files
…

## How to test:
1. check into current branch
2. do `npm install` and `npm start` to run this PR locally
3. go to localhost:3000
4. check if you see logo in the navbar and logo images in menu items where real image is not available.
5. Once you've checked the above, merge this PR into development and after a few minutes check [hungry-rabbit.onrender.com/](https://hungry-rabbit.onrender.com/). If the website has been updated, autodeployment is on!
